.travis.yml

```diff
@@ -29,27 +29,23 @@ deploy:
   - provider: script
     skip_cleanup: true
     script: >
-     bash -c '
-     if [ "${ARCH}" == "amd64" ]; then
-       docker push nfvpe/sriov-device-plugin;
-     fi
-     docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$ARCH;
-     docker push nfvpe/sriov-device-plugin:$ARCH;
-     echo done'
+      if [ "${TARGET}" == "amd64" ]; then
+        docker push nfvpe/sriov-device-plugin;
+      fi
+      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$ARCH;
+      docker push nfvpe/sriov-device-plugin:$ARCH;
     on:
       branch: master
   # Push image to Dockerhub on tag
   - provider: script
     skip_cleanup: true
     script: >
-      bash -c '
       if [ "${ARCH}" == "amd64" ]; then
-        docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:"$TRAVIS_TAG";
-        docker push nfvpe/sriov-device-plugin:"$TRAVIS_TAG";
+        docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$TRAVIS_TAG;
+        docker push nfvpe/sriov-device-plugin:$TRAVIS_TAG;
       fi
-      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
-      docker push nfvpe/sriov-device-plugin:"$TRAVIS_TAG-$ARCH";
-      echo done'
+      docker tag nfvpe/sriov-device-plugin nfvpe/sriov-device-plugin:$TRAVIS_TAG-$ARCH;
+      docker push nfvpe/sriov-device-plugin:$TRAVIS_TAG-$ARCH;
     on:
       tags: true
       all_branches: true
```

Comment:
@ahalim-intel I think you can test if you setup a Travis CI on your forked repo, this line show it should run the deploy section if the PR is pushed on any branch.